x-pack/legacy/plugins/integrations_manager/server/integrations/data.ts

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I think a few simple comments on top of each of the exported functions in this file to explain what they do in relationship to each other would go a long way in helping new folks understand the flow here, and then we can massage the actual names of the functions as we better understand how it all fits together or what we need for other cases, etc. Otherwise this is all really clear and clean.

Reply:
Agreed. Breaking this up helped me see some patterns. 

If it’s OK with you, I’d like to hold off on that until I add another asset type (e.g. https://github.com/elastic/kibana/issues/38609) because I think that might influence things. Things don’t need to be “final” before I add the comments, just looking to get back to making after this refactoring then see how things fit together.